tenant/config/crds/tenancy_v1alpha1_tenant.yaml

```diff
@@ -27,6 +27,11 @@ spec:
         metadata:
           type: object
         spec:
+          properties:
+            tenantAdminNamespaceName:
```

Comment:
Thoughts on calling it `tenantParentNamespaceName` instead? I could go either way, just seeing what others think.

Reply:
In a hierarchical structure, "parent" is ambiguous term which may indicate many nodes. Here this namespace acts more like a "root" so I think tenantAdminNamespaceName sounds better.

Reply:
Right, but the work @adrianludwin is doing may mean we have nested tenants. It's not something we've really discussed.